resources/views/users/Quiz.blade.php

```diff
@@ -10,7 +10,7 @@
 <div class='container'>
```

Comment:
In this file please change file name. Must use camelCase format for blade name not StudlyCase format